utils/generate_dotnet_channels.js

```diff
@@ -227,4 +230,10 @@ function processCustomType(type, optional) {
     && inlineType(type.properties.name).ts === 'string') {
     return { ts: 'DeviceDescriptorEntry', scheme: 'tObject()', optional };
   }
-}
\ No newline at end of file
+}
+
+function rewriteChannelType(type) {
+  if (['BrowserType'].includes(type))
```

Comment:
Before it generated it to `Core.Playwright` which works, with the current generator it leads to just `BrowserType` which conflicts with [this](https://github.com/microsoft/playwright-dotnet/blob/f1067ec2d20b4061ef0a60a96a2d144f254e3e12/src/Playwright/BrowserType.cs) BrowserType instead of [this](https://github.com/microsoft/playwright-dotnet/blob/8728aced9db702205ed175ad180ee66d603eec4b/src/Playwright/Core/BrowserType.cs).

So we need to rewrite it back to the more precise import.

Reply:
Let's do `Core.Foo` everywhere and remove using namespace.